Xamarin.Forms.Platform.GTK/Cells/CellBase.cs

```diff
@@ -0,0 +1,150 @@
+﻿using Gtk;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Linq;
+
+namespace Xamarin.Forms.Platform.GTK.Cells
+{
+    public abstract class CellBase : EventBox
```

Comment:
We don't want to make lot's of thigns public, i maybe recomend have a Internals namespace at least

Reply:
Yeah, good one. Changed native cell controls to become internal classes.

Reply:
Out of curiosity, is there a reason to keep so many things internal ? From the lib client point of view, it often makes it painful to customize a renderer behaviour